### PR TITLE
Updates to patient model

### PR DESCRIPTION
**Pull request by upasanapattnaik-eq**
As part of [CNDIT 1245](url): Updates to the patient model to point to SRTE value. The codes are required as part of the ElasticSearch payload, the value for the codes are used by reporting. This update will effect the output of nrt_patient topic. 

Updates: 
+ Stored Procedure Model
+ Reporting Model

### CODE REVIEW

**Review by rseshadri99 — APPROVED** [review]
@upasanapattnaik-eq looks good. Please send a diff. PR for Stored Proc.

File: data-reporting-service/person-service/src/main/resources/application.yaml [hunk 1]
```diff
@@ -11,16 +11,16 @@ spring:
           topic-name: nbs_default
       output:
         providerElastic:
-          topic-name: ${PROVIDER_ELASTIC:nrt_provider_elastic}
+          topic-name: ${PROVIDER_ELASTIC:elastic_nrt_provider}
```
Comment (alog-enquizit) [inline data-reporting-service/person-service/src/main/resources/application.yaml:14]: I also made the same change in my deployment branch. Will have to resolve the merge conflicts between us :)

File: data-reporting-service/person-service/src/main/java/gov/cdc/etldatapipeline/person/transformer/PersonTransformers.java [hunk 2]
```diff
@@ -112,28 +112,28 @@ public PatientReporting buildPatientReporting(PatientSp p) {
                 .addlGenderInfo(p.getAdditionalGenderCd())
                 .addUserId(p.getAddUserId())
                 .ageReported(p.getAgeReported())
-                .ageReportedUnit(p.getAgeReportedUnitCd())
+                .ageReportedUnit(p.getAgeReportedUnit())
```
[the remark this answers is not in the document]
Comment (upasanapattnaik-eq) [reply data-reporting-service/person-service/src/main/java/gov/cdc/etldatapipeline/person/transformer/PersonTransformers.java:115]: Yes, this is only required for Patient Reporting. The code is being called as part of ElasticSearch.

File: data-reporting-service/person-service/src/main/resources/application.yaml (hunk 1 above)
Comment (upasanapattnaik-eq) [reply data-reporting-service/person-service/src/main/resources/application.yaml:14]: No problem! I am testing something that might need further changes in the model. We can discuss the convention offline.

**Review by alog-enquizit — APPROVED** [review]
lgtm 👍

**Comment by upasanapattnaik-eq** [comment]
Thank you for reviewing! Will add the sproc updates to a different PR.

**Pull request merged**